config/mesh_defaults.sh

```diff
@@ -156,6 +156,34 @@ case $GRIDNAME in
       FORECASTWALLTIME="07:00:00" # forecast wall clock time
       # FIXME: no unit offset url
       ;;
+   "LAERDCv5j")
+      #
```

Comment:
comment `#` needed?

Reply:
Will do. This is the successor to the LAv20a mesh (Louisiana). v5j is the version number (they elected not to use the year in the version number unfortunately). Not sure why ERDC is in the file name, that's how we received it.